### PR TITLE
Child org tab information added to Accounts page

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/manage/manage-subscription/create-manage-orgs-service-providers.md [hunk 1]
```diff
@@ -139,9 +139,39 @@ Provisioning Cloud SIEM can take up to 24 hours. You can determine provisioning
...
+<img src={useBaseUrl('img/subscriptions/account-overview-new.png')} alt="your description" style={{border:'1px solid black'}} />
+
 ### View child credit usage by category
 
-You can view the usage by child orgs on the **Account Overview** page. Usage for the following categories is shown:
+You can view the usage of individual child orgs in the **Child Orgs** tab on the **Account Overview** page. Usage for the following categories is shown:
+
+* The **Credit Allocation** tile indicates the total number of credits allocated.
+* The **Total Credit Usage** tile shows amount of credits consumed out of the total allocated credits.
+* The **Usage Forecast** tile indicated the total consumption of the credits by the end of contract period according to the current consumption rate.
```
Comment (jpipkin1) [inline docs/manage/manage-subscription/create-manage-orgs-service-providers.md:165]:
```suggestion
* The **Usage Forecast** tile indicates the total consumption of the credits by the end of contract period according to the current consumption rate.
```

File: docs/manage/manage-subscription/create-manage-orgs.md [hunk 2]
```diff
@@ -121,14 +121,45 @@ You can view the usage by child orgs on the **Account Overview** page. Usage for
...
-### View cross-org credits allocation and consumption
+<img src={useBaseUrl('img/subscriptions/account-overview-new.png')} alt="your description" style={{border:'1px solid black'}} />
+
+### View child credit usage by category
+
+You can view the usage of individual child orgs in the **Child Orgs** tab on the **Account Overview** page. Usage for the following categories is shown:
+
+* The **Credit Allocation** tile indicates the total number of credits allocated.
+* The **Total Credit Usage** tile shows amount of credits consumed out of the total allocated credits.
+* The **Usage Forecast** tile indicated the total consumption of the credits by the end of contract period according to the current consumption rate.
```
Comment (jpipkin1) [inline docs/manage/manage-subscription/create-manage-orgs.md:133]:
```suggestion
* The **Usage Forecast** tile indicates the total consumption of the credits by the end of contract period according to the current consumption rate.
```

**Review by jpipkin1 — APPROVED** [review]
Looks great, @JV0812!

File: docs/manage/manage-subscription/create-manage-orgs-service-providers.md [hunk 3]
```diff
@@ -139,9 +139,39 @@ Provisioning Cloud SIEM can take up to 24 hours. You can determine provisioning
...
+* The **Allocated Credits** column indicates the total number of credits allocated to the selected child org.
+* The **Total Credits Used** column indicates the total amount of credits consumed for the selected child org.
+* The **Usage %** column indicates the percentage of overall subscription credits that was allocated to a child org has been consumed.
+* The **Forecast** column indicates the percentage of total consumption of the credits by the end of contract period according to the current consumption rate for a selected child org.
+* The **Usage % WOW** column indicates the percentage of week on week subscription credits that was allocated to a child org has been consumed.
+
+<img src={useBaseUrl('img/subscriptions/child-org.png')} alt="your description" style={{border:'1px solid black'}} />
+
+On clicking any of the selected Child Org, you can view the usage for the following categories:
```
Comment (kimsauce) [inline docs/manage/manage-subscription/create-manage-orgs-service-providers.md:174]:
```suggestion
By clicking any of the selected child orgs, you can view the usage for the following categories:
```

**Pull request merged**